app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,44 @@
+class QuestionsController < ApplicationController
+
+  def index
+    @questions = Question.where(user_id: current_user.id).order("created_at DESC")
+  end
+
+  def show
+    @question = Question.find params[:id]
+    @action = question_answers_path(@question)
+    @answer = Answer.new
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def create
+    params[:question][:user_id] = current_user.id
+    @question = Question.new question_params
+    if @question.save
+      send_notification(@question.content, "New Question")
+      redirect_to question_path(@question)
+    else
+      flash[:error] = 'Field cannot be left blank.'
+      redirect_to new_question_path
+    end
+  end
+
+  def edit
+  end
+
+  def update
+  end
```

Comment:
In rails, you can remove empty actions like this - rails will still render the associated view.